docs/user-guide.md

```diff
@@ -24,26 +24,26 @@ that might be null then you write `@Nullable String x;`.
 
 ## Types and nullness
 
-In the JSpecify world, most references have one of three possible properties
+In the JSpecify world, most TODO references have one of three possible properties
 regarding nullness:
 1. They can be null.
 2. They can't be null.
 3. We don't know if they can be null or not.
 
-If `x` is a reference of a certain type, then if `x` can be null, `x.toString()` is
+If `x` is a reference of a certain type, then if `x` can be null, the dereference `x.toString()` is
 unsafe because it could produce a NullPointerException. If `x` can't be null,
 `x.toString()` can never produce a NullPointerException. And if we don't know
 whether `x` can be null or not, `x.toString()` might or might not be unsafe.
 
 There are two JSpecify annotations that indicate these properties:
 * `@Nullable` applied to a type denotes a reference of that type that can be null.
-* `@NullMarked` applied to a module, class, or package means that a reference in
+* `@NullMarked` applied to a class, package, or module means that a reference in
```

Comment:
Reordered to be consistent.

Reply:
(How would standardizing it on the opposite order be? The wider the scope the better.... e.g. I think we may be adding "method", but I wouldn't want it to be highly emphasized.)